src/services/doppler-api-client.double.ts

```diff
@@ -72,4 +101,22 @@ export class HardcodedDopplerApiClient implements DopplerApiClient {
     //   error: new Error('Dummy error'),
     // };
   }
+
+  public async getSubscriberSentCampaigns(
+    email: string,
+    apikey: string,
+  ): Promise<ResultWithoutExpectedErrors<CampaignDeliveryCollection>> {
+    console.log('getApiSubscriber');
```

Comment:
Here it should be `console.log('getSubscriberSentCampaigns');` right?

Reply:
thanks!